libyul/optimiser/Suite.cpp

```diff
@@ -127,5 +128,9 @@ void OptimiserSuite::run(
 	Rematerialiser::run(_dialect, ast);
 	UnusedPruner::runUntilStabilised(_dialect, ast);
 
+	SSAReverser{}(ast);
```

Comment:
I'm not sure where best to put this... this is probably not the best place, but running it earlier doesn't seem to work well either...

Reply:
Added it to the loop before the expression inliner, replacing an SSATransform - notably that change doesn't seem to affect anything currently, but as we said: might still make sense to have it there.